spec/controllers/sign_in/application_controller_spec.rb

```diff
@@ -113,6 +113,34 @@ def service_account_auth
       end
     end
 
+    shared_context 'mpi profile validation' do
+      before { allow_any_instance_of(SignIn::UserLoader).to receive(:find_valid_user).and_return(nil) }
```

Comment:
I don't think you need this line? I don't see it on the other mpi profile validation block at least

Reply:
nvm I tried removing it and I see errors